src/lib/forms/components/LanguageField.js

```diff
@@ -4,14 +4,15 @@ import { VocabularyField } from '@forms/core/VocabularyField';
 
 export class LanguageField extends Component {
   render() {
-    const { multiple, type, fieldPath } = this.props;
+    const { multiple, type, fieldPath, required } = this.props;
     return (
       <VocabularyField
         multiple={multiple}
         type={type}
         fieldPath={fieldPath}
         label={multiple ? 'Languages' : 'Language'}
         placeholder={multiple ? 'Select languages...' : 'Select language'}
+        required={required}
```

Comment:
I have doubts about the languages being required. In case of a library having only books in one language, f.e. english it is unnecessary @ntarocco